GomokuClient/fix-sdk-gen.js

```diff
@@ -0,0 +1,19 @@
+console.log("Fixing sdk.gen.ts...");
+
+const fs = require("fs");
+const path = require("path");
+const sdkFilePath = path.resolve(
+  __dirname,
+  "packages/gomoku-api/client/sdk.gen.ts",
+);
+
+let content = fs.readFileSync(sdkFilePath, "utf8");
+
+content = content.replace(
+  /"Content-Type": "application\/json",\s*\.\.\.options\?\.headers,/g,
+  '...options?.headers, "Content-Type": "application/json",',
+);
+
+fs.writeFileSync(sdkFilePath, content, "utf8");
+
+console.log("Fixed redundant 'Content-Type' declarations in sdk.gen.ts.");
```

Comment:
This is not super good to have, but a workaround to satisfy current typecheck, potentially we have to remove on server side explicit conte-type

```shell
➜  packages git:(chore/upgrade-packages) ✗ yarn typecheck
turbo 2.3.3

• Packages in scope: @gomoku/core, @gomoku/story
• Running typecheck in 2 packages
• Remote caching disabled
@gomoku/core:typecheck: cache miss, executing de8109231cb411f6
@gomoku/story:typecheck: cache miss, executing 9e1cf6f4133f0b26
@gomoku/core:typecheck: 
@gomoku/story:typecheck: 
@gomoku/core:typecheck: ../gomoku-api/client/sdk.gen.ts:132:7 - error TS2783: 'Content-Type' is specified more than once, so this usage will be overwritten.
@gomoku/core:typecheck: 
@gomoku/core:typecheck: 132       "Content-Type": "application/json",
@gomoku/core:typecheck:           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
@gomoku/core:typecheck: 
@gomoku/core:typecheck:   ../gomoku-api/client/sdk.gen.ts:133:7
@gomoku/core:typecheck:     133       ...options?.headers,
@gomoku/core:typecheck:               ~~~~~~~~~~~~~~~~~~~
@gomoku/core:typecheck:     This spread always overwrites this property.
@gomoku/core:typecheck: 
@gomoku/core:typecheck: ../gomoku-api/client/sdk.gen.ts:155:7 - error TS2783: 'Content-Type' is specified more than once, so this usage will be overwritten.
@gomoku/core:typecheck: 
@gomoku/core:typecheck: 155       "Content-Type": "application/json",
@gomoku/core:typecheck:           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
@gomoku/core:typecheck: 
@gomoku/core:typecheck:   ../gomoku-api/client/sdk.gen.ts:156:7
@gomoku/core:typecheck:     156       ...options?.headers,
@gomoku/core:typecheck:               ~~~~~~~~~~~~~~~~~~~
@gomoku/core:typecheck:     This spread always overwrites this property.
@gomoku/core:typecheck: 
@gomoku/core:typecheck: ../gomoku-api/client/sdk.gen.ts:264:7 - error TS2783: 'Content-Type' is specified more than once, so this usage will be overwritten.
@gomoku/core:typecheck: 
@gomoku/core:typecheck: 264       "Content-Type": "application/json",
@gomoku/core:typecheck:           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
@gomoku/core:typecheck: 
@gomoku/core:typecheck:   ../gomoku-api/client/sdk.gen.ts:265:7
@gomoku/core:typecheck:     265       ...options?.headers,
@gomoku/core:typecheck:               ~~~~~~~~~~~~~~~~~~~
@gomoku/core:typecheck:     This spread always overwrites this property.
@gomoku/core:typecheck: 
@gomoku/core:typecheck: 
@gomoku/core:typecheck: Found 3 errors.
@gomoku/core:typecheck: 
@gomoku/core:typecheck: ERROR: command finished with error: command (/Users/kuprins/Desktop/Coding/gomoku/gomoku/GomokuClient/packages/gomoku-core) /private/var/folders/1l/ym7pjxzs1hz_m634vdyjgs8w0000gn/T/xfs-09476ffb/yarn run typecheck exited (1)
@gomoku/story:typecheck: 
@gomoku/core#typecheck: command (/Users/kuprins/Desktop/Coding/gomoku/gomoku/GomokuClient/packages/gomoku-core) /private/var/folders/1l/ym7pjxzs1hz_m634vdyjgs8w0000gn/T/xfs-09476ffb/yarn run typecheck exited (1)

 Tasks:    0 successful, 2 total
Cached:    0 cached, 2 total
  Time:    2.473s 
Failed:    @gomoku/core#typecheck

 ERROR  run failed: command  exited (1)
➜  packages git:(chore/upgrade-packages) ✗ cat gomoku-core/tsconfig.json 
{
  "compilerOptions": {
    "composite": true
  },
  "references": [
    { "path": "./tsconfig.app.json" },
    { "path": "./tsconfig.node.json" }
  ],
  "exclude": ["dist", "node_modules", "lingui.config.ts", "../gomoku-api", "@gomoku/api"],
  "include": []
}
➜  packages git:(chore/upgrade-packages) ✗ 


```

Reply:
Created -> https://github.com/ligomoku/gomoku/issues/265